torchtune/utils/_import_guard.py

```diff
@@ -5,11 +5,23 @@
 # LICENSE file in the root directory of this source tree.
 
 import torch
-from torchtune.utils._version import torch_version_ge
+from torchtune.utils._version import (
+    _get_torchao_version,
+    _is_fbcode,
+    _nightly_version_ge,
+    torch_version_ge,
+)
 
 # We can only use flex attention / BlockMask if torch version >= 2.5.0 and GPU is Turing / SM75 and above
 _SUPPORTS_FLEX_ATTENTION = (
     torch_version_ge("2.5.0")
     and torch.cuda.is_available()
     and torch.cuda.get_device_capability() >= (7, 5)
 )
+
+torchao_version = _get_torchao_version()
```

Comment:
couldn't this return None if is_fbcode? a bit awkward to return either a version string or None, but htne you check for fbcode again below

Reply:
Yeah good point. Honestly I have half a mind to just scrap this util altogether now and use `torchao.__version__` directly

Reply:
Yeah good point. Just did a check and in fbcode `torchao.__version__ == 'unknown'`, so it at least won't throw an error. Then I can just explicitly say `ao_version = torchao.__version__` in `_import_guard.py`, gate behind `_is_fbcode()` in `_USE_NEW_TENSOR_CORE_TILED_LAYOUT_API`, and delete `_get_torchao_version` altogether